contracts/TestERC20.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// @unsupported: ovm
```

Comment:
This should cause Optimism-solc compiler to ignore this file but it does not for unknown reasons